core/impl/src/main/java/pt/webdetails/cdf/dd/util/Utils.java

```diff
@@ -172,12 +172,12 @@ public static Document getDocument( InputStream input ) throws DocumentException
 
   @SuppressWarnings( "unchecked" )
   public static List<Element> selectNodes( Element elem, String xpath ) {
-    return elem.selectNodes( xpath );
+    return (List<Element>) (List<?>) elem.selectNodes( xpath );
```

Comment:
Please, rename this method (and its overload) to `selectElements`.
You can remove the overload, if you simply declare the first argument as `Node`.
Then, use this method instead of directly using `node.selectNodes`, above, in `XmlFsPluginModelReader`.